server/chat-plugins/othermetas.ts

```diff
@@ -542,4 +542,57 @@ export const commands: ChatCommands = {
 	crossevolvehelp: [
 		"/crossevo <base pokemon>, <evolved pokemon> - Shows the type and stats for the Cross Evolved Pokemon.",
 	],
+
+	'!showevo': true,
+	showevo(target) {
+		if (!this.runBroadcast()) return;
+		const targetid = toID(target);
+		if (!targetid) return this.parse('/help showevo');
+		const evo = Dex.getSpecies(target);
+		if (!evo.exists) {
+			return this.errorReply(`Error: Pokemon not found.`);
```

Comment:
```suggestion
			return this.errorReply(`Error: Pok\u00e9mon not found.`);
```
I would also make this state the target so people can get their typo pointed out to them

Reply:
makes sense